sled-agent/types/src/instance.rs

```diff
@@ -31,15 +29,17 @@ pub struct InstanceEnsureBody {
     /// state this sled agent should store for this incarnation of the instance.
     pub hardware: InstanceHardware,
 
-    /// The instance runtime state for the instance being registered.
-    pub instance_runtime: InstanceRuntimeState,
-
     /// The initial VMM runtime state for the VMM being registered.
     pub vmm_runtime: VmmRuntimeState,
 
     /// The ID of the instance for which this VMM is being created.
     pub instance_id: InstanceUuid,
 
+    /// The ID of the migration in to this VMM, if this VMM is being
+    /// ensured s part of a migration in. If this is `None`, the VMM is not
```

Comment:
```suggestion
    /// ensured is part of a migration in. If this is `None`, the VMM is not
```

Reply:
hahahaha the typo fix made the OpenAPI document invalid and broke CI, that's so cool: https://buildomat.eng.oxide.computer/wg/0/details/01J7HBCTJXDT1JGJM8ZEE8VYFD/sYGwteiE5lgB6Rwrljh8k3aeNx9Uy36Qmafr9z7bozCKXH5x/01J7HBDCZYCHGYV1ZSGBJ60266#S5834